Asterix/optics/propagation_functions.py

```diff
@@ -3,21 +3,54 @@
 
 
 def mft(image,
-        real_dim_input,
-        dim_output,
-        nbres,
+        real_dim_input=4,
+        dim_output=4,
+        nbres=1,
         inverse=False,
         norm='backward',
         X_offset_input=0,
         Y_offset_input=0,
         X_offset_output=0,
-        Y_offset_output=0):
-    """
-    Based on Matrix Direct Fourier transform (MFT) from R. Galicher
-    (cf. Soummer et al. 2007, OSA)
+        Y_offset_output=0,
+        only_mat_mult=False,
+        AA=None,
+        BB=None,
+        norm0=None,
+        returnAABB=False):
+    """Based on Matrix Direct Fourier transform (MFT) from R. Galicher (cf.
+    Soummer et al. 2007, OSA).
+
         - Return the Matrix Direct Fourier transform (MFT) of a 2D image
         - Can deal with any size, any position of the 0-frequency...
 
+    This function measures 2 matrices AA and BB, and the norm norm0. The MFT itself is
+    the matrix multiplication norm0 * ((AA @ image) @ BB) (@ is matrix multiplication)
```

Comment:
"(@ is matrix multiplication)" --> ", where @ is a matrix multiplication"

Reply:
done